mongo/collection.go

```diff
@@ -1493,9 +1483,7 @@ func (coll *Collection) FindOneAndDelete(ctx context.Context, filter interface{}
 		}
 		op = op.Comment(comment)
 	}
-	if fod.MaxTime != nil {
-		op = op.MaxTimeMS(int64(*fod.MaxTime / time.Millisecond))
-	}
+	op = op.MaxTime(fod.MaxTime)
```

Comment:
Since we are no longer wrapping this condition in an if block, can we just extend the operation builder [here](https://github.com/mongodb/mongo-go-driver/blob/6e9fc900d992c6aa53d5004e6c75ba1bb994f2b4/mongo/collection.go#L1475) ?